### PR TITLE
Added sitemap

### PR DESCRIPTION
**Pull request by maceq687**

### CODE REVIEW

File: robots.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Allow: /
+
+Sitemap: https://www.deranged.dk/sitemap.xml
```
Comment (hypesystem) [inline robots.txt]: Missing newline @ EOF

File: sitemap.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<url>
+<loc>https://deranged.dk/explainers/</loc>
+</url>
+<url>
+<loc>https://deranged.dk/files/team-cv.pdf</loc>
+</url>
+<url>
+<loc>https://deranged.dk/explainers/small-deliveries</loc>
+</url>
+</urlset>
```
Comment (hypesystem) [inline sitemap.xml]: Missing newline @ EOF

Would be nice with some indentation.

For elements with many attributes you can do this:

```xml
<urlset
    xmlns="..."
    xmlns:xsi="..."
   ...
>
```

File: sitemap.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+</url>
+<url>
+<loc>https://deranged.dk/privacy/</loc>
+</url>
+<url>
+<loc>https://deranged.dk/explainers/</loc>
+</url>
+<url>
+<loc>https://deranged.dk/files/team-cv.pdf</loc>
+</url>
```
Comment (hypesystem) [inline sitemap.xml]: Can remove this

**Review by hypesystem — APPROVED** [review]
I'm worried about maintaning this manually... I feel like there must be away to automate it a bit more.

File: robots.txt (hunk 1 above)
Comment (maceq687) [reply robots.txt]: Fixed in [bb0f63a](https://github.com/derangeddk/deranged.dk/pull/102/commits/bb0f63aad1c7f1cf170b2043c79c8d78389f4802)

File: sitemap.xml (hunk 3 above)
Comment (maceq687) [reply sitemap.xml]: Removed in [864234a](https://github.com/derangeddk/deranged.dk/pull/102/commits/864234a3e90d5e4172b72ae140b0785c5b2262a9)

File: sitemap.xml (hunk 2 above)
Comment (maceq687) [reply sitemap.xml]: Fixed in [fdaadf6](https://github.com/derangeddk/deranged.dk/pull/102/commits/fdaadf6aa18aa5c1c650eba2dd387ee709c188b9)

**Pull request merged**